addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientDetailsEdit.js

```diff
@@ -10,7 +10,11 @@ odoo.define('point_of_sale.ClientDetailsEdit', function(require) {
         constructor() {
             super(...arguments);
             this.intFields = ['country_id', 'state_id', 'property_product_pricelist'];
-            this.changes = {};
+            const partner = arguments[1].partner;
```

Comment:
```suggestion
            const partner = this.props.partner;
```

Hi Ivan. Thank you for this fix. But before merging, can you consider this suggestion? This is because `partner` is passed as `props` (via `t-props="state.editModeProps"`) to this component.

Reply:
done